data/ka'het/ka'het missions.txt

```diff
@@ -99,56 +120,45 @@ mission "First Contact: Ka'het: Remnant 1B"
-		spaceport `	Incidentally, when the invasion broke out, Osaeli was also the world closest to the Ka'het fleets: it was bombarded and devastated for years until every Builder on the planet was killed.`
+		description `	For most of the thousands of years that passed between the time their discovery of spaceflight and their extinction, Osaeli was the de facto capital of the Builders, a key part of their economy and society. The first Het station was constructed orbiting this planet's star, with the purpose of protecting them from any external threat.`
 	planet Palarei
-		spaceport `Thanks to their earlier exploration vessels, this was where the Builders encountered the first lifeforms that evolved outside of their home planet. While the climate was drastically different for the Builders, the fruits the local flora produced were popular enough that Palarei became a major farming planet.`
-		spaceport `	The farms on Palarei were one of the few successful attempts at creating fully automatic facilities. Partly thanks to the rudimental repair systems they developed, a few dozen of them are still working even today.`
+		description `	Thanks to their earlier exploration vessels, this was where the Builders encountered the first lifeforms that evolved outside of their home planet. While the climate was drastically different for the Builders, the fruits the local flora produced were popular enough that Palarei became a major farming planet.`
 	planet Parii'het
-		spaceport `Parii'het was almost finished when the invasion broke out, and already capable of (slowly) creating Ka'het. Thanks to the mining operations on Alix and the occasional asteroid mining operated by the Mar drones, most of the facilities are still fully operational to this day.`
-		spaceport `	Different from most of the Graveyard, the station doesn't seem to have ever suffered any damage from the Ka'het, nor from the Builders trying to regain control over it.`
+		description `	Parii'het was almost finished when the invasion broke out, and already capable of creating Ka'het. Thanks to the mining operations on Alix and the occasional asteroid mining operated by the Mar unmanned vessels, most of the facilities are still fully operational to this day.`
 	planet Retilie
-		spaceport `Retilie was used by the builders mainly for fishing, leading to a lack of infrastructural development. The few settlements on the planet were never as advanced as the ones on the nearby worlds, and when the Ka'het arrived there was basically nothing to stop them: after a few attacks, the planet was uninhabited again.`
-		spaceport `	In the oceans, the many species of fish continue to prosper, unaware of the destruction the planet has undergone.`
+		description `	As it was mainly used for fishing, the settlements on this planet were never as advanced as the ones on the nearby worlds, and when the Ka'het arrived there was basically nothing to stop them: after a few attacks, the planet was uninhabited again.`
```

Comment:
```suggestion
		description `	As it was mainly used for fishing, the settlements on this planet were never as advanced as the ones on nearby worlds, so when the Ka'het arrived, there was basically nothing to stop them: after a few attacks, the planet was uninhabited again.`
```
OR
```suggestion
		description `	As it was mainly used for fishing, the settlements on this planet were never as advanced as the ones on the nearby worlds, and when the Ka'het arrived there was virtually nothing to stop them: after a few attacks, the planet was uninhabited again.`
```
`Basically` vs `virtually` is personal preference. Wanted to make two suggestions since there were grammar fixes that needed to happen regardless of whether you preferred that word choice.